src/OpenSage.Game/Content/ContentManager.cs

```diff
@@ -79,12 +79,25 @@ public sealed class ContentManager : DisposableBase
 
             IniDataContext = new IniDataContext(fileSystem);
 
-            IniDataContext.LoadIniFiles(@"Data\INI\Default\Object.ini");
-            IniDataContext.LoadIniFiles(@"Data\INI\Object");
-            IniDataContext.LoadIniFiles(@"Data\INI\multiplayer.ini");
-            IniDataContext.LoadIniFile(@"Data\INI\PlayerTemplate.ini");
-
-            IniDataContext.LoadIniFile(@"maps\MapCache.ini");
```

Comment:
All of those were loaded too early (before GameData.ini) which resulted in errors.